test/spoke/integration/SyncDeposit.t.sol

```diff
@@ -200,4 +202,42 @@ contract SyncDepositTest is SyncDepositTestHelper {
         vm.expectRevert(ISyncManager.ExceedsMaxMint.selector);
         syncVault.mint(1, self);
     }
+
+    // --- erc165 checks ---
+    function testERC165SupportSyncDeposit(bytes4 unsupportedInterfaceId) public {
+        bytes4 erc165 = 0x01ffc9a7;
+        bytes4 erc7575Vault = 0x2f0a18c5;
+        bytes4 asyncVaultOperator = 0xe3bc4e65;
+        bytes4 asyncVaultRedeem = 0x620ee8e4;
+        bytes4 asyncVaultCancelRedeem = 0xe76cffc7;
+        bytes4 erc7741 = 0xa9e50872;
+        bytes4 erc7714 = 0x78d77ecb;
+
+        vm.assume(
+            unsupportedInterfaceId != erc165 && unsupportedInterfaceId != erc7575Vault
+                && unsupportedInterfaceId != asyncVaultOperator && unsupportedInterfaceId != asyncVaultRedeem
+                && unsupportedInterfaceId != asyncVaultCancelRedeem && unsupportedInterfaceId != erc7741
+                && unsupportedInterfaceId != erc7714
+        );
```

Comment:
Ultra NIT (optional): Add on top: `// forgefmt: disable-next-item` to improve readability